lib/dgram.js

```diff
@@ -710,8 +682,74 @@ Socket.prototype.getSendBufferSize = function() {
 };
 
 
+// Deprecated private APIs.
+Object.defineProperty(Socket.prototype, '_handle', {
+  get: util.deprecate(function() {
+    console.log(this);
```

Comment:
why is the `console.log(this)` added?

Reply:
Because I forgot to remove it *facepalm*